simde/x86/sse4.2.h

```diff
@@ -92,6 +92,231 @@ SIMDE_BEGIN_DECLS_
   #define _SIDD_UNIT_MASK SIMDE_SIDD_UNIT_MASK
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+int
+simde_mm_cmpestra_8_(simde__m128i a, int la, simde__m128i b, int lb, const int imm8)
```

Comment:
Either the native aliases script needs to not match on trailing `[A-Za-z0-9_]` or this function could be renamed `simde_x_mm_cmpestra_8_` to avoid the issue

Reply:
The former I think.  I'll take a look at the native aliases renamer.

Meanwhile, I think a better solution would be to use `simde_mm_cmpestra()` in the tests; it's the public API we're interested in testing anyways.  They can still be in test separate functions, _mm_cmpestra will delegate to the right function based on imm8.

The trailing _ indicates that it's a private API; _x is for public SIMDe extensions, so I think the name should stay.